app/views/indicators/index.html.erb

```diff
@@ -94,41 +77,55 @@
                    other_class: '-no-wrap',
                    column_key: 'definition' %>
       </div>
+      <div class="small-2 columns">
+        <%= render 'shared/components/c_order_filter_button',
+                   text: 'Type',
+                   other_class: '-no-wrap',
+                   column_key: 'type' %>
+      </div>
     </div>
   </div>
   <% @indicators.each do |indicator| %>
     <div class="c-table-list-item">
       <div class="row">
         <div class="small-2 columns">
-          <div class="c-table-list-item__text f-ff1-m-bold"><%= link_to indicator.alias, model_indicator_url(@model, indicator) %></div>
-        </div>
-        <div class="small-2 columns">
-          <div class="c-table-list-item__text f-ff1-m"><%= indicator.category %></div>
+          <div class="c-table-list-item__text f-ff1-m-bold"><%= indicator.name %></div>
         </div>
         <div class="small-2 columns">
-          <div class="c-table-list-item__text f-ff1-m"><%= indicator.subcategory %></div>
-        </div>
-        <div class="small-1 columns">
-          <div class="c-table-list-item__text f-ff1-m"><%= indicator.name %></div>
-        </div>
-        <div class="small-1 columns">
           <div class="c-table-list-item__text f-ff1-m"><%= indicator.unit %></div>
         </div>
         <div class="small-3 columns">
           <div class="c-table-list-item__text f-ff1-m">
             <%= indicator.definition && truncate(indicator.definition, length: 120) %>
           </div>
         </div>
-        <div class="small-1 columns">
+        <div class="small-2 columns">
+          <div class="c-table-list-item__text f-ff1-m">
+            <% if indicator.parent_id.present? %>
+              <% parent_indicator = Indicator.find(indicator.parent_id) %>
```

Comment:
We can avoid the `find` here it by using `indicator.parent`. To eliminate an extra query to fetch the parent, we can use `Indicator.includes(:parent)` in `Indicator#fetch_all`